packages/react/src/components/media/controls/images/OfflineStreamImage.tsx

```diff
@@ -0,0 +1,29 @@
+const OfflineStreamImage = () => (
+  <div
+    dangerouslySetInnerHTML={{
```

Comment:
We should replace this with just the svg element content, instead of using `dangerouslySetInnerHTML`

Reply:
```tsx
const OfflineStreamImage = () => (<svg>...</svg>)
```

Reply:
I wrapped this this way because just copying and pasting the svg, then we have react asking to rename some properties (`fill-rule` to `fillRule` etc) otherwise it will throw an error. Anyway, I have now renamed all the props. Which we have to redo in case we update the image.